app/plugin/TreeColumnInResolutionRange.js

```diff
@@ -0,0 +1,157 @@
+        var layer = this;
+        return layer.id === record.getOlLayer().id;
+    },
+
+    /**
+     * When resolution changes in map: update tree nodes if needed
+     * @param {ol.Object.Event} evt The openlayers event
+     */
+    onChangeResolution: function (evt) {
+        var me = this;
+        var mapView = evt.target;
+        var unit = mapView.getProjection().getUnits();
+        var resolution = mapView.getResolution();
+        var treepanel = me.cmp.up('treepanel');
+        var nodeStore = treepanel.getStore();
+        var treeNodes = nodeStore.getData();
+        Ext.each(treeNodes.items, function (node) {
+            var inRange = me.layerInResolutionRange(node.getOlLayer(), resolution);
+            node[inRange ? 'removeCls' : 'addCls']('cpsi-tree-node-disabled');
+            var descriptionBefore = node.getOlLayer().get('description');
+            var description = me.getTooltipText(node, inRange, unit);
+            var changed = description !== descriptionBefore;
+            node.getOlLayer().set('description', description);
+            if (changed) {
+                node.triggerUIUpdate();
+            }
+        });
+        // This triggers the rendering if any existing StyleSwitcherRadioGroups
+        treepanel.fireEvent('itemupdate');
+    },
+
+    /**
+     * Check if layer is visible in current resolution range
+     *
+     * @param {ol.layer.Base} layer The OpenLayers layer
+     * @param {Number} currentRes The current map resolution
+     *
+     * @returns {Boolean} layer is in resolution range?
+     */
+    layerInResolutionRange: function (layer, currentRes) {
+        if (!layer || !currentRes) {
+            // It is questionable what we should return in this case, I opted for
+            // false, since we cannot sanely determine a correct answer.
+            return false;
+        }
+        var layerMinRes = layer.getMinResolution(); // default: 0 if unset
+        var layerMaxRes = layer.getMaxResolution(); // default: Infinity if unset
+        // minimum resolution is inclusive, maximum resolution exclusive
+        var within = currentRes >= layerMinRes && currentRes < layerMaxRes;
+        return within;
+    },
+
+    /**
+     * Get tooltip text for node
+     *
+     * @param {Ext.data.NodeInterface} node The layer tree node
+     * @param {Boolean} inRange Is layer in range
+     * @param {String} unit The unit defined in current map projection
+     *
+     * @returns {String} The toolip text valid for current range
+     */
+    getTooltipText: function (node, inRange, unit) {
+        if (!Ext.isDefined(node.originalQtip)) {
+            node.originalQtip = node.getOlLayer().get('description') || '';
+        }
+
+        if (inRange) {
+            return node.originalQtip.length > 0 ? node.originalQtip : undefined;
+        }
+
+        var currentTip = node.getOlLayer().get('description');
+        if (currentTip && (currentTip.indexOf('<br />') > -1 ||
```

Comment:
Maybe a more robust selector than `'<br />'` would be good here.

Reply:
See new issue https://github.com/compassinformatics/cpsi-mapview/issues/124